hibernate-core/src/main/java/org/hibernate/id/IdentifierGenerator.java

```diff
@@ -60,4 +59,13 @@
 	 * @throws HibernateException Indicates trouble generating the identifier
 	 */
 	Serializable generate(SharedSessionContractImplementor session, Object object) throws HibernateException;
+
+	/**
+	 * Check if JDBC batch inserts are supported.
+	 *
+	 * @return JDBC batch inserts are supported.
+	 */
+	default boolean supportsJdbcBatchInserts() {
+		return true;
+	}
```

Comment:
@sebersole could you check this out? I think it's cleaner to have a proper contract rather than relying on an `instanceof IdentityGenerator` but I would like to be sure it's not an issue to add this method here.

Reply:
With the default, I think its fine to add

Reply:
I always prefer a "proper contract"

Reply:
To make it clear, was it ok to merge it or you wanted to have it implemented differently?

Reply:
The implementation is great.  I was just agreeing with @gsmet

Reply:
I agree. It's much cleaner now.